cantabular-import/README.md

```diff
@@ -0,0 +1,36 @@
+### Cantabular Import Journey ###
+
+## Requirements ##
+
+Expects you to have environment variables `zebedee_root` and 
+`SERVICE_AUTH_TOKEN` set in your local environment
+
+Expects your services to be in the expected relative path
```

Comment:
I had to change the path to Zebedee in zebedee.yml, because I have it cloned somewhere else.
Maybe we can add: `otherwise, you will need to change the context paths in the corresponding componse files`?

Reply:
Yeah interesting to see how many people have things cloned to different places.. I would think this file should use however most people have it..